buildbot-conf/cordova-config.json.sample

```diff
@@ -0,0 +1,43 @@
+{
+    "couchdb": {
+        "uri":  "http://localhost:5984"
+    },
+    "dashboard": {
+        "port": 8088
+    },
+    "app": {
+        "entry":   "autotest/pages/all.html",
+        "release": "master",
+        "timeout": 600
```

Comment:
Can we have `platforms` parameter back? So people could filter platforms / builders they want to run on medic.